web-playground/e2e/sample-tester.test.ts

```diff
@@ -38,7 +38,15 @@ page.on("dialog", (dialog) => {
 
 describe('test all grammars with puppeteer', () => {
     beforeEach(async () => {
+        // Forward the console log from the browser
+        page.on('console', msg => console.log('PAGE LOG:', msg.text()));
+
         await page.goto('http://localhost:3000/playground');
+
+        // Test without on-the-fly formatting, because the debounce makes things
+        // less explicit and predictable.
+        const checkboxEl = await page.waitForSelector("#onTheFlyFormatting") ?? fail('Did not find checkbox');
+        await checkboxEl.click();
```

Comment:
If I understand correctly, this disables on-the-fly formatting because the checkbox is ticked by default. This seems a bit error-prone: if we ever choose to disable on-the-fly formatting by default, this will enable it instead. Is there a `checkbox.disable()` instead?